kv/bucket.go

```diff
@@ -361,7 +359,6 @@ func (s *Service) FindBuckets(ctx context.Context, filter influxdb.BucketFilter,
 
 	if needsSystemBuckets {
```

Comment:
Here is also legacy, so everything from:

```go
needsSystemBuckets := true
for _, b := range bs {
```

through to line 382 can be deleted.